packages/contract-schema/spec/abi.spec.json

```diff
@@ -85,9 +88,12 @@
           "$ref": "#/definitions/StateMutability"
         },
         "constant": { "type": "boolean" },
```

Comment:
Should `constant` be deleted here?  Like there's no way it should ever occur here, right?

Reply:
I'm not sure what the behavior used to be in older solc - do you know offhand?

Reply:
Hm, checking the docs it might actually produce it after all.  I'm actually not sure offhand, worth checking.

Reply:
Compiling this:

```solidity
pragma solidity ^0.4.0;

contract Foo {
  function Foo () public {
  }
}
```

With:

```console
$ npx solc@0.4.4 --abi ./Foo.sol
 ```

I get:

```json
[{"inputs":[],"type":"constructor"}]
```

I can't seem to get `constant` to appear in the constructor ABI. But I'm not sure we want to remove it from the spec here? Especially if the docs suggest that it's allowable. WDYT?

Reply:
OK yeah (considering the point you raised on Slack about someone could write it by hand) I guess it makes sense to not change this.